### PR TITLE
Desktop version of portfolio website

### PR DESCRIPTION
**Pull request by tanveerisonline**
### Following is the list of changes.

1. Added media queries so this portfolio website can work also on desktops.
2. Made it responsive using media queries.
3. followed the Figma design completely.
4. It can work beautifully also on mobile phones and with any browser.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,20 +5,35 @@
...
-      <nav>
+      <nav class="desktop-navbar">
+        <div class="desktop-nav">
+          <a href="#" target="_parent" class="logo">Tanveer Ahmad</a>          
+        </div>
+        <div class="nav-links">  
+          <ul>
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
```
Comment (aimalamiri) [inline index.html]: Great job! but it's required to follow the original Figma design, so I please make sure to use the same color for your navbar menu links. 

## Original
image

## Yours
image

File: index.html [hunk 2]
```diff
@@ -31,22 +46,48 @@ <h1 class="author">I’m Tanveer Glad to see you!</h1>
             <li><a href="#"><img src="./assets/Vector-2.svg" alt="m-icon"></a></li>
             <li><a href="#"><img src="./assets/Vector-1.svg" alt="github"></a></li>
             <li><a href="#"><img src="./assets/Vector-3.svg" alt="victory"></a></li>
-          </ul>
+          </ul>
         </div>
+      </div>
```
Comment (aimalamiri) [inline index.html]: The design is fantastic, and you can improve it by following the same Figma design rules. So kindly make sure the bottom-left corner of the header is rounded.

image

File: index.html [hunk 3]
```diff
@@ -139,55 +184,61 @@ <h2 class="title">Multi-Post Stories</h2>
...
+                  <li><a href="#"><img src="./assets/Vector-3.svg" alt="victory"></a></li>
+                </ul>
+              </div>
+          
+              <div class="action">
+                <button type="button" class="default_button">Get my resume</button>
+              </div>
+          </div>
+    </div>
+      <div class="languages">
```
Comment (aimalamiri) [inline index.html]: Good job! but as you know it's required to follow the original Figma design, so please make sure to consider the below points.
1. The top-right corner of the about section is rounded
2. You have some extra space to the right in the **Languages** section collapse. Please remove it.

image

**Review by aimalamiri — CHANGES_REQUESTED** [review]
Hi @tanveerisonline, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlmostThereReachGIF

## Highlights

- The desktop version is designed very well and look fantastic ✔️ ⭐
- Good job for using the right GitHub Flow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are properly set up and passing 🟢 👍

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -1,242 +1,466 @@
...
+            </ul>
+            <div class="frameworks-skills">
+              <p class="subheader">Frameworks</p>
+              <img src="images/side-arrow.png" alt="An arrow pointing right" />
+            </div>
+
+            <div class="frameworks-skills">
+              <p class="subheader">Skills</p>
+              <img src="images/side-arrow.png" alt="An arrow pointing right" />
+            </div>
```
Comment (Lizdev-05) [inline index.html]: [OPTIONAL] Great job @tanveerisonline 👍. Kindly check your image path, the arrow is not showing.
arro

**Review by Lizdev-05 — CHANGES_REQUESTED** [review]
Hi @tanveerisonline, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.



 ## Highlights
- The desktop version is designed very well and looks fantastic ✔️ ⭐
- Good job for using the right GitHub Flow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are properly set up and passing 🟢 👍

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (tanveerisonline) [reply index.html]: hi but i have checked my project in 4 different browsers and it is working fine may be from your side their is some issue

**Review by BregornOriginal — APPROVED** [review]
# Hi @tanveerisonline 👍🏼 ,

## Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳 🥳 🥳

## Congratulations! 🥇 

## Good work applying the changes requested by the previous reviewers. Now it's time to take some rest and be ready for the next week!! 💯 💯 💯

## Highlights

✅ Linters are passing
✅ Good use of GitHub Flow
✅ Keep improving your PR description! 💯 

905cc73355510c2245a9f7d2993f09c8_w200

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Pull request merged**